doc/arkode/guide/source/Mathematics.rst

```diff
@@ -2074,3 +2074,51 @@ step attempt, or fails with the minimum step size, then the integration is halte
 and an error is returned. In this case the user may need to employ other
 strategies as discussed in :numref:`ARKODE.Usage.ARKStep.Tolerances` and
 :numref:`ARKODE.Usage.ERKStep.Tolerances` to satisfy the inequality constraints.
+
+.. _ARKODE.Mathematics.Relaxation:
+
+Relaxation Methods
+==================
+
+For cases where the problem :eq:`ARKODE_IVP` is conservative or dissipative with
+respect to a smooth convex function :math:`\xi(y(t))`, it is desirable to have
+the numerical method preserve these properties. That is
+:math:`\xi(y_n) = \xi(y_{n-1}) = \ldots = \xi(y_{0})` for conservative problems
+and :math:`\xi(y_n) \leq \xi(y_{n-1})` for dissipative problems. To this end,
+ARKODE supports relaxation methods
+:cite:p:`ketcheson2019relaxation, kang2022entropy, ranocha2020relaxation` to
+ensure dissipation or preservation of a global function.
+
+Relaxation methods compute a new solution
+
+.. math::
+   y_r = y_{n-1} + r d = r y_n + (1 - r) y_{n - 1}
+   :label: ARKODE_RELAX_SOL
+
+where :math:`d \equiv h_n \sum_{i=1}^{s}(b^E_i f^E_i + b^I_i f^I_i)` is the update
+direction and the relaxation factor, :math:`r`, is selected to ensure
+conservation or dissipation. Given a second order method with
+:math:`b^E_i \geq 0` and :math:`b^I_i \geq 0`, the factor :math:`r` is computed
+by solving the auxiliary scalar nonlinear system
+
+.. math::
+   F(r) = \xi(y_{n-1} + r d) - \xi(y_{n-1}) - r e = 0
+   :label: ARKODE_RELAX_NLS
+
+at the end of each time step. The estimated change in :math:`\xi` is given by
+:math:`e \equiv h_n \sum_{i=1}^{s} \langle \xi'(z_i), b^E_i f^E_i + b^I_i f^I_i \rangle`
+where :math:`\xi'` is the Jacobian of :math:`\xi`.
+
+By default the nonlinear system :eq:`ARKODE_RELAX_NLS` is solved using Newton
+iteration. Optionally, a fixed-point iteration of Brent's method can be
+utilized. If this iteration fail to meet the specified tolerances in the
+maximum allowed number of iterations, the step size is reduced by the factor
+:math:`\eta_\text{rf}` (default 0.25) and the step is repeated. Additionally, a
+relaxation value greater than :math:`r_\text{max}` (default 0.8) or less than
+:math:`r_\text{min}` (default 1.2), will be considered as a failed solve and the
```

Comment:
I "unresolved" this conversation only because in the GitHub API I cannot make a new comment/suggestion on the preceding two lines.

Here, the values of 0.8 and 1.2 are accidentally swapped (it lists 0.8 as the maximum and 1.2 as the minimum).

Reply:
I must have fixed this some time between this comment and the most recent version because I'm not seeing these swapped.